pkg/client/openshift/buildconfig.go

```diff
@@ -101,6 +106,38 @@ func (b *buildConfig) TriggerBuild(bc *buildv1.BuildConfig, triggedBy string) (b
 	return true, nil
 }
 
+// TriggerFromFileBuild wil be called by kogito-cli when a build from file is performed.
+// When called a new build will be triggered with the request kogito resource or a tgz file.
+func (b *buildConfig) TriggerFromFileBuild(namespace string, r io.Reader, options *buildv1.BinaryBuildRequestOptions) (*buildv1.Build, error) {
```

Comment:
Can you please extend the name of `b`, `bc` and `r`? These shortcuts complicates reading (at least for me).

Reply:
b is comming from the same class, buildCOnfig struct, same for all other methods.
will update the others.